compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -116,6 +116,35 @@ extern "C" LLVMValueRef LLVMRustGetNamedValue(LLVMModuleRef M, const char *Name,
   return wrap(unwrap(M)->getNamedValue(StringRef(Name, NameLen)));
 }
 
+enum class LLVMRustTailCallKind {
+  None,
+  Tail,
+  MustTail,
+  NoTail,
+  Last,
+};
+
+static CallInst::TailCallKind fromRust(LLVMRustTailCallKind Kind) {
+  switch (Kind) {
+  case LLVMRustTailCallKind::None:
+    return CallInst::TailCallKind::TCK_None;
+  case LLVMRustTailCallKind::Tail:
+    return CallInst::TailCallKind::TCK_Tail;
+  case LLVMRustTailCallKind::MustTail:
+    return CallInst::TailCallKind::TCK_MustTail;
+  case LLVMRustTailCallKind::NoTail:
+    return CallInst::TailCallKind::TCK_NoTail;
+  case LLVMRustTailCallKind::Last:
+    return CallInst::TailCallKind::TCK_Last;
```

Comment:
```suggestion
    return CallInst::TailCallKind::TCK_LAST;
```
Accordingly to https://llvm.org/doxygen/classllvm_1_1CallInst.html#ad682514c13f12f1a8d759d422fce6aef it's `TCX_LAST` (not `TCX_Last`)

Reply:
Ah.